src/prefect/utilities/importtools.py

```diff
@@ -25,4 +25,7 @@ def import_object(name: str) -> Any:
         mod = importlib.import_module(mod_name)
         return getattr(mod, attr_name)
     except ValueError:
-        return importlib.import_module(name)
+        pass
```

Comment:
This raised an error instead of returning as intended 🤦

Reply:
maybe not worth doing in this PR, but a little refactor of this code to make it more clear what we're doing would be nice

it's not clear at first glance that `name.rsplit(".",1)` is the line expected to throw a `ValueError`

Reply:
Yeah I was confused as well.. I was hoping my comment was helpful. Pass this one to the fuuuture